modules/op-tkn-pipeline-management.adoc

```diff
@@ -16,45 +16,45 @@ $ tkn pipeline --help
 
 == pipeline delete
 
-Delete a Pipeline.
+Delete a pipeline.
 
-.Example: Delete the `mypipeline` Pipeline from a namespace
+.Example: Delete the `mypipeline` pipeline from a namespace
 [source,terminal]
 ----
 $ tkn pipeline delete mypipeline -n myspace
 ----
 
 == pipeline describe
-Describe a Pipeline.
+Describe a pipeline.
 
-.Example: Describe `mypipeline` Pipeline
+.Example: Describe `mypipeline` pipeline
 [source,terminal]
 ----
 $ tkn pipeline describe mypipeline
 ----
 
 == pipeline list
-List Pipelines.
+List pipelines.
 
-.Example: Display a list of Pipelines
+.Example: Display a list of pipelines
 [source,terminal]
 -----
 $ tkn pipeline list
 -----
 
 == pipeline logs
-Display Pipeline logs for a specific Pipeline.
+Display pipeline logs for a specific pipeline.
 
-.Example: Stream live logs for the `mypipeline` Pipeline
+.Example: Stream live logs for the `mypipeline` pipeline
 [source,terminal]
 ----
 $ tkn pipeline logs -f mypipeline
 ----
 
 == pipeline start
-Start a Pipeline.
+Start a pipeline.
 
-.Example: Start `mypipeline` Pipeline
+.Example: Start `mypipeline` pipeline
```

Comment:
Please review the IBM SG section on "Articles," which states "Articles increase clarity and ease translation. The articles “a” and “an” are indefinite articles. Use them to refer to things in the abstract or not in the particular. “An offering” or “a product” means any offering or any product, as opposed to a specific offering or product. “The” is a definite article. Use it to refer to a particular thing or things, meaning that “the offering” or “the product” is about that particular offering or product. Do not omit the articles a, an, and the."

Please review and update all of these revisions accordingly.

Reply:
```suggestion
.Example: Start the `mypipeline` pipeline
```